README.md

```diff
@@ -1 +1,290 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [My Capstone Project] <a name="about-project"></a>
+
+ This project is to put to test the understanding of individual student on what they've studied so far from the curriculum's module 1.!
+ - Here is the page preview
+
+> |     | Mobile version                                  | Desktop version                                  |     |
+> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
+> |     | ![Screenshot1](./images/screenshots/mobile-view.png) | ![Screenshot2](./images/screenshots/desktop1.png) |
+
+
+
+> Learning objectives
+
+- I Used semantic HTML tags.
+- I Applied best practices in HTML code.
+- Used CSS selectors correctly.
+- Used CSS box model.
+- Used Flexbox to place elements on the page.
+- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
+- Used GitHub Pages to deploy web pages.
+- Applied JavaScript best practices and language style guides in code.
+- Used JavaScript to manipulate DOM elements.
+- Used JavaScript events.
+- Used objects to store and access data.
+- Communicated technical concepts to other technical people.
+
+ 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Git&Github</summary>
+  <ul>
+    <li><a href="https://docs.github.com/en/get-started/using-git/about-git">Git&Github</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Javascript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+## Additional Tools
+
+- WebHint
+- Stylelint
+- ESlint
+- Node Packet Manager
+
+<!-- Features -->
+
+
+### Key Features <a name="key-features"></a>
+
+
+- [Create the toolbar]
+- [Create the headline section]
+- [Import svg images and style accordingly]
+- [Enhancing CSS classes ]
+- [Use css minifier to get on css file ]
+- [buttons]
+- [Contact Form]
+- [Navigation Section]
+- [Portfolio Page]
+- [Headline Section]
+- [About page]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+https://aj193.github.io/Capstone-project/index.html
+
+## 🚀 Loom Link <a name="Loom-Link"></a>
+https://www.loom.com/share/24614a0619f2470b8388d46accd262bf
```

Comment:
- [ ] Great job so far @AJ193  :muscle: You did great adding the video presentation of your project which contained a clear and objective description of the project as well as a demo of the project, showing the pages in different screen sizes. However, as the previous reviewer mentioned, it is missing a highlight about a piece of the code you find interesting or are proud of making as stated in the [Project Requiements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md). So kindly include that in your presentation :100: